docs/GroupDefs.hpp

```diff
@@ -48,6 +48,42 @@
 /*!
  * \defgroup ControlSystemGroup Control System
  * \brief Contains control system elements
+ *
+ * The control system manages the time-dependent mapping between the fixed
```

Comment:
How about "...between frames, such as the fixed...".  Because we will also have a "distorted frame".